ribs/visualize.py

```diff
@@ -675,32 +675,33 @@ def parallel_axes_plot(archive,
                          f"but is '{cbar_orientation}'")
 
     # If there is no order specified, plot in increasing numerical order.
-    if bc_order is None:
+    if measure_order is None:
         cols = np.arange(archive.behavior_dim)
         axis_labels = [f"behavior_{i}" for i in range(archive.behavior_dim)]
         lower_bounds = archive.lower_bounds
         upper_bounds = archive.upper_bounds
 
-    # Use the requested behaviors (may be less than the original number of bcs).
+    # Use the requested measures (may be less than the original number of
+    # measures).
     else:
         # Check for errors in specification.
-        if all(isinstance(bc, int) for bc in bc_order):
-            cols = np.array(bc_order)
+        if all(isinstance(measure, int) for measure in measure_order):
+            cols = np.array(measure_order)
             axis_labels = [f"behavior_{i}" for i in cols]
         elif all(
-                len(bc) == 2 and isinstance(bc[0], int) and
-                isinstance(bc[1], str) for bc in bc_order):
-            cols, axis_labels = zip(*bc_order)
+                len(measure) == 2 and isinstance(measure[0], int) and
+                isinstance(measure[1], str) for measure in measure_order):
+            cols, axis_labels = zip(*measure_order)
             cols = np.array(cols)
         else:
-            raise TypeError("bc_order must be a list of ints or a list of"
+            raise TypeError("measure_order must be a list of ints or a list of"
                             "tuples in the form (int, str)")
 
         if np.max(cols) >= archive.behavior_dim:
             raise ValueError(f"Invalid Behavior: requested behavior index "
                              f"{np.max(cols)}, but archive only has "
                              f"{archive.behavior_dim} behaviors.")
-        if any(bc < 0 for bc in cols):
+        if any(measure < 0 for measure in cols):
```

Comment:
measures

Reply:
measure. also, remove "Behavior" in the ValueError below this line.